From a8f530020e3fc06ca7963111e1ced890ecb596e2 Mon Sep 17 00:00:00 2001
From: CAB233 <yidaduizuoye@outlook.com>
Date: Sat, 15 Mar 2025 23:10:25 +0800
Subject: [PATCH] clean BUILD_ARGS

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index dbd7fad..03bf4d1 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ else
 	VERSION ?= unstable-$(date).r$(count).$(commit)
 endif
 
-BUILD_ARGS := -trimpath -ldflags "-s -w -X github.com/daeuniverse/dae/cmd.Version=$(VERSION) -X github.com/daeuniverse/dae/common/consts.MaxMatchSetLen_=$(MAX_MATCH_SET_LEN)" $(BUILD_ARGS)
+BUILD_ARGS := $(BUILD_ARGS)
 
 .PHONY: clean-ebpf ebpf dae submodule submodules
 
-- 
2.48.1

